astropy/modeling/projections.py

```diff
@@ -64,7 +64,7 @@
 projcodes = [code for _, code in _PROJ_NAME_CODE]
 
 
-__all__ = [
+__all__ = [  # noqa: PLE0605
```

Comment:
Instead of this, please split the `__all__` to

```
__all__ = [...]
__all__ += + list(map("_".join, product(["Pix2Sky", "Sky2Pix"], chain(*_PROJ_NAME_CODE))))  # noqa: PLE0605
```

Reply:
@WilliamJamieson, do you want to keep this dynamic or actually enumerate the list for static purposes?